programming/index.md

```diff
@@ -22,6 +22,10 @@ Our code is versioned using [Git](https://git-scm.com/). Git lets us collaborate
 - [Brief Introduction](https://try.github.io)
 - [Branching and Code Review](branching)
 
+## Vision Tracking
+How to use the opencv library and the basics of vision tracking and camera callibration
```

Comment:
- Can you capitalize OpenCV and link to https://opencv.org/
- Calibration, not callibration